src/commands/RunCommand.js

```diff
@@ -12,7 +13,20 @@ export default class RunCommand extends Command {
       return;
     }
 
-    this.packagesWithScript = this.packages
+    let packagesToRunCommandIn = this.packages;
+
+    if (this.flags.restrictTo && `${this.flags.restrictTo}`.length > 0) {
```

Comment:
I would consider `--restrict-to` without anything specified to be invalid input with an error that says so